Abilities/npc_dota_hero_skywrath_mage/skywrath_mage_concussive_shot.cs

```diff
@@ -18,7 +23,7 @@ public float Radius
         {
             get
             {
-                return this.Ability.GetAbilitySpecialData("launch_radius");
+                return this.Ability.GetAbilitySpecialData("launch_radius") - 25f;  // bug fixed
```

Comment:
this is probably the hull radius which should be subtracted when calculating the size rather than modifying the return value here?

Reply:
hull radius more than 25 gives 💯 
for work 25 was enough